backend/utils/data.py

```diff
@@ -3,122 +3,116 @@
 # %%
 import pandas as pd
 import numpy as np
-import plotly
-import plotly.graph_objs as go
 
-def getData(area):
+def getData(area: str) -> pd.DataFrame:
   # %%
-  AREA = area
-  processedData = pd.DataFrame()
-
-  AHS = pd.read_csv("data/ACS-Average-Household-Size.tsv", sep='\t')
-  H = pd.read_csv("data/ACS-Households.tsv", sep='\t')
-  MHI = pd.read_csv("data/ACS-Median-Household-Income.tsv", sep='\t')
-  PB2 = pd.read_csv("data/ACS-Pop-below-200.tsv", sep='\t')
-  P = pd.read_csv("data/ACS-Population.tsv", sep='\t')
-  LEAPP = pd.read_csv("data/CDHS-Leap-Participants.tsv", sep='\t')
-  EAP = pd.read_csv("data/EOC-EA-Participants.tsv", sep='\t')
-  PES = pd.read_csv("data/EOC-Primary-Energy-Source.tsv", sep='\t')
-  UT = pd.read_csv("data/EOC-Utility-Territories.tsv", sep='\t')
-  XEAP = pd.read_csv("data/EOC-Xcel-EA-Participants.tsv", sep='\t')
+  AREA: str = area
+  processedData: pd.DataFrame = pd.DataFrame()
```

Comment:
The variable `processedData` doesn't end up getting used, as it gets set at line 115.

Reply:
Removed that as it is no longer needed.